.rubocop.yml

```diff
@@ -0,0 +1,129 @@
+AllCops:
+  DisplayCopNames: true
+  TargetRubyVersion: 2.3
```

Comment:
NB: 2.3 is the minimal Ruby version modern Rubocop supports.
Instead of trying to find the old version that will support some older Rubies (down to 1.8?..), I just turned off the suggestions that are brought because of the version choice (safe navigation operator, modern hashes, `%i[]` literals etc.)